hack/test-end-to-end.sh

```diff
@@ -156,9 +156,9 @@ echo "[INFO] Volumes dir is:            ${VOLUME_DIR}"
 echo "[INFO] Certs dir is:              ${CERT_DIR}"
 
 # Start All-in-one server and wait for health
-# Specify the scheme and port for the master, but let the IP auto-discover
+# Specify the scheme and port for the listen address, but let the IP auto-discover.  Set --public-master to localhost, for a stable link to the console.
 echo "[INFO] Starting OpenShift server"
-sudo env "PATH=${PATH}" openshift start --listen="${API_SCHEME}://0.0.0.0:${API_PORT}" --hostname="127.0.0.1" --volume-dir="${VOLUME_DIR}" --etcd-dir="${ETCD_DATA_DIR}" --cert-dir="${CERT_DIR}" --loglevel=4 &> "${LOG_DIR}/openshift.log" &
+sudo env "PATH=${PATH}" openshift start --listen="${API_SCHEME}://0.0.0.0:${API_PORT}" --public-master="${API_SCHEME}://localhost" --hostname="127.0.0.1" --volume-dir="${VOLUME_DIR}" --etcd-dir="${ETCD_DATA_DIR}" --cert-dir="${CERT_DIR}" --loglevel=4 &> "${LOG_DIR}/openshift.log" &
```

Comment:
Setting public master to localhost may break builds and other components that need to talk back to the master?